README.md

```diff
@@ -263,6 +263,40 @@ spec:
     name: cluster-creds
 ```
 
+The `vars` field supports HCL string, number, bool, object and list types. For example, the following variable can be populated using the accompanying Terraform spec:
+
+```hcl
+variable "cluster_spec" {
+  type = object({
+      region = string
+      env = string
+      node_count = number
+      public = bool
+  })
+}
+```
+
+```yaml
+apiVersion: infra.contrib.fluxcd.io/v1alpha1
+kind: Terraform
+metadata:
+  name: helloworld
+  namespace: flux-system
+spec:
+  approvePlan: "auto"
+  path: ./
+  sourceRef:
+    kind: GitRepository
+    name: helloworld
+    namespace: flux-system
+  vars:
+    cluster_spec:
```

Comment:
From reading the codes,  I think this example should be?
```
spec:
  approvePlan: "auto"
  path: ./
  sourceRef:
    kind: GitRepository
    name: helloworld
    namespace: flux-system
  vars:
  - name: cluster_sepc
    value:
      region: us-east-1
      env: dev
      node_count: 10
      public: false    
```

Reply:
Yup, thanks for catching @chanwit.

Reply:
Looks like the older example was wrong also; both now fixed.